types/webpack/index.d.ts

```diff
@@ -560,7 +560,7 @@ declare namespace webpack {
             /** Assign modules to a cache group */
             test?: ((...args: any[]) => boolean) | string | RegExp;
             /** Select chunks for determining cache group content (defaults to \"initial\", \"initial\" and \"all\" requires adding these chunks to the HTML) */
-            chunks?: "initial" | "async" | "all";
+            chunks?: "initial" | "async" | "all" | ((chunk: any) => boolean);
```

Comment:
I just checked and it seems like support for this function might've been added after webpack 4.1 – not sure of the best way to handle this 🤔

Reply:
@mohsen1 Just updated the version to 4.4 – it may have been introduced earlier than 4.4.0, but 4.4.1 is the version I'm working with locally so I'm more comfortable jumping straight to 4.4. Lemme know if you think it'd be better to use the version this was introduced in instead, though!

Reply:
`Chunk` is typed. Can you replace `any` with it here?

Reply:
Oh damn, good catch – just updated!